src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const nextValue = function(n) {
+    const amount = result;
```

Comment:
you only need it if there is no `n`, so move it to if block, don't create unnecessary variables